pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dependencies:
   path: ^1.8.2
   win32: ^5.1.1
   cross_file: ^0.3.3+7
+  js: ^0.7.1
```

Comment:
Is that intended?
It's not used anymore and doesn't work with WASM anyways :)

Reply:
it is a mistake, I removed it I don’t use it anyway
thanks